benchmarks/asv.conf.json

```diff
@@ -0,0 +1,85 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "pymc3",
+
+    // The project's homepage
+    "project_url": "https://pymc-devs.github.io/pymc3/",
+
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    "repo": "..",
+
+    // List of branches to benchmark. If not provided, defaults to "master"
+    // (for git) or "tip" (for mercurial).
+    "branches": ["master"],
+
+    // The DVCS being used.  If not set, it will be automatically
+    // determined from "repo" by looking at the protocol in the URL
+    // (if remote), or by looking for special directories, such as
+    // ".git" (if local).
+    "dvcs": "git",
+
+    // The tool to use to create environments.  May be "conda",
+    // "virtualenv" or other value depending on the plugins in use.
+    // If missing or the empty string, the tool will be automatically
+    // determined by looking for tools on the PATH environment
+    // variable.
+    "environment_type": "virtualenv",
```

Comment:
Do the PyMC devs have a strong preference here? I haven't setup the benchmark server with virtualenv, so `conda` would be slightly easier for me. Though it probably isn't much work to get virtualenv working too.

Reply:
I think `virtualenv` is my preference, everyone else uses `conda`, and I'm happy to go there as well!